aopy/visualization/base.py

```diff
@@ -321,6 +321,7 @@ def plot_ECoG244_data_map(data, bad_elec=[], interp=True, cmap='bwr', ax=None, *
         bad_elec (list, optional): channels to remove from the plot. Defaults to [].
         interp (bool, optional): flag to include 2D interpolation of the result. Defaults to True.
         cmap (str, optional): matplotlib colormap to use in image. Defaults to 'bwr'.
+        theta (float, optional): rotation in degrees to apply to channel mapping. Default 0.
```

Comment:
Some comments about the direction of rotation.  
```suggestion
        theta (float, optional): rotation in degrees to apply to channel mapping. Default 0. rotations are applied clockwise. Ex: , theta = 90 rotates the map clockwise by 90 degrees, -90 rotates the map anti-clockwise by 90 degrees.
```